runtime/plugin/osc52.lua

```diff
@@ -29,6 +29,10 @@ vim.api.nvim_create_autocmd('UIEnter', {
       return
     end
 
+    if os.getenv('ConEmuHooks') == 'Enabled' then
```

Comment:
why not add this to the condition on line 25 ?

Reply:
No strong reason, I wasn't sure what is the preferred practice in neovim.
I tried to separate concerns.

I merged the test into the one at line 10.